source/common/network/connection_impl.cc

```diff
@@ -1037,7 +1037,11 @@ void ClientConnectionImpl::connect() {
   } else {
     immediate_error_event_ = ConnectionEvent::RemoteClose;
     connecting_ = false;
-    setFailureReason(absl::StrCat("immediate connect error: ", errorDetails(result.errno_)));
+    setFailureReason(absl::StrCat(
+        "immediate connect error: ", errorDetails(result.errno_), "remote address family:",
```

Comment:
Probably want a space or some other delimiter at the start of `"remote address family:"` or it and the remote error will run together.